app/src/components/ArticleItem.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { Component } from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import { createStyles, WithStyles } from '@material-ui/styles';
+import classNames from 'classnames';
+
+import Img from 'react-image';
+import {
+    Typography,
+    Box,
+    ButtonBase,
+    Modal,
+    Backdrop,
+    Fade,
+    CircularProgress,
+} from '@material-ui/core';
+import OpenInNewIcon from '@material-ui/icons/OpenInNew';
+import { Link } from 'react-router-dom';
+import bigInt from 'big-integer';
+import { Styles } from '../styles/ArticleItem';
+
+import {
+    Article,
+} from '../models/NewsTimeline';
+import { INSTR_COLOR, NANOSECONDS_IN_ONE_SECOND } from '../constants/Constants';
+import { getHoursMinutesStringFromTimestamp } from '../utils/date-utils';
+import ArticleDetails from './ArticleDetails';
+
+const styles = createStyles(Styles);
+
+interface Props extends WithStyles<typeof styles>{
+    article: Article,
+    isFirstItem: boolean,
+}
+
+interface State {
+    modalIsOpen: boolean,
+}
+
+class ArticleItem extends Component<Props, State> {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            modalIsOpen: false,
+        };
+    }
+
+    /**
+     * @desc Handles closing the modal
+     */
+    handleCloseModal = () => {
+        this.setState({
+            modalIsOpen: false,
+        });
+    };
+
+    render() {
+        const { article, classes, isFirstItem } = this.props;
+        const { modalIsOpen } = this.state;
+        const nanosecondTimestamp = bigInt(article.timestamp).multiply(NANOSECONDS_IN_ONE_SECOND);
+        const timePublishedString = getHoursMinutesStringFromTimestamp(nanosecondTimestamp);
+
+        const instruments = article.tickers.filter(ticker => {
+            return ticker in INSTR_COLOR;
+        });
```

Comment:
We want all tickers to appear not just the ones we have on the backend

Reply:
Sorry about that, I thought we didn't want to include them. For most articles, the number of tickers was pretty big, so I thought it looked cluttered. I'll start working on this.

Reply:
How big does it get?

Reply:
i think you can get up to 6-7 at times, making the line cluttered with instrument pills.

Reply:
I just pushed a commit @chimano, you can see all the tickers now if you want to see for yourself

Reply:
Looks good!